ivy/functional/frontends/builtins/func_wrapper.py

```diff
@@ -0,0 +1,101 @@
+import functools
+from typing import Callable, Iterable
+from importlib import import_module
+
+import ivy
+
+
+def _to_ivy_array(x):
+    # if x is any frontend array including frontend list
+    # then extract the wrapped ivy array
+    if hasattr(x, "ivy_array"):
+        return x.ivy_array
+
+    # convert native arrays and lists to ivy arrays
+    elif isinstance(x, ivy.NativeArray):
```

Comment:
If we want lists to get converted, we have to change this to `else`.

Reply:
Yeah, I hust forgot to modify the comment.